src/sentry/api/endpoints/organization_projects_experiment.py

```diff
@@ -0,0 +1,188 @@
+import logging
+import random
+import string
+
+from django.db import IntegrityError, transaction
+from rest_framework.exceptions import PermissionDenied
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.serializers import ValidationError
+
+from sentry import audit_log, features
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationPermission
+from sentry.api.endpoints.team_projects import ProjectSerializer
+from sentry.api.exceptions import ConflictError, ResourceDoesNotExist
+from sentry.api.serializers import serialize
+from sentry.experiments import manager as expt_manager
+from sentry.models import Project
+from sentry.models.organization import Organization
+from sentry.models.organizationmember import OrganizationMember
+from sentry.models.organizationmemberteam import OrganizationMemberTeam
+from sentry.models.team import Team
+from sentry.signals import project_created, team_created
+from sentry.utils.snowflake import MaxSnowflakeRetryError
+
+CONFLICTING_TEAM_SLUG_ERROR = "A team with this slug already exists."
+MISSING_PERMISSION_ERROR_STRING = "You do not have permission to join a new team as a Team Admin."
+
+
+def generate_three_letter_string():
+    letters = string.ascii_lowercase
+    return "".join(random.choice(letters) for _ in range(3))
+
+
+# This endpoint is intended to be available to all members of an
+# organization so we include "project:read" in the POST scopes.
+
+
+class OrgProjectPermission(OrganizationPermission):
+    scope_map = {
```

Comment:
Just double checking

Are these scopes inclusive or exclusive?
If we have `project:admin` here, won't that still make this available only to members with admin privs?

Reply:
The scopes are inclusive so as long as you have one of them you can access the endpoint